CHANGELOG.md

```diff
@@ -2,6 +2,14 @@
 
 Records breaking changes from major version bumps
 
+## 10.0.0
+
+The `search_filters.yml` manifest has been renamed to `<doc type>_search_filters.yml`. Code which referenced a
```

Comment:
This is a very backward not-compatible change. I guess it makes sense, but we'll need a buyer app PR and it would be reassuring to know you've checked this isn't going to be horrendous for some reason :)

Reply:
I think I remember you telling me "this is only ever referenced from the buyer app so it should be fine"...

Reply:
Yeah yeah, it's definitely only the buyer app. Hopefully there's only one place :)